From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chris Bobbe <cbobbe@zulip.com>
Date: Fri, 1 Sep 2023 18:21:09 -0700
Subject: [tsflower] Import TurboModule type using path that works

I think TsFlower's `subst/react-native` could handle this, with
something like (untested):

export type { TurboModule } from 'react-native/Libraries/TurboModule/RCTExport';
---
 .../lib/typescript/src/specs/NativeSafeAreaContext.js.flow      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git types/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaContext.js.flow types/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaContext.js.flow
index 9d8051c08..22525e663 100644
--- types/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaContext.js.flow
+++ types/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaContext.js.flow
@@ -1,7 +1,7 @@
 /* @flow
  * @generated by TsFlower
  */
-import { type TurboModule } from 'react-native';
+import { type TurboModule } from 'react-native/Libraries/TurboModule/RCTExport';
 import { type Double } from 'react-native/Libraries/Types/CodegenTypes';
 
 export interface Spec extends TurboModule {
-- 
2.32.0

